### PR TITLE
feat(web): digest onboarding workflow analytics

### PR DESCRIPTION
**Pull request by LetItRock**
### What change does this PR introduce?

Digest Onboarding Workflow Analytics.
Tested with Segment Debugger.

### Other information (Screenshots)
Screenshot 2023-03-17 at 18 28 12

Screenshot 2023-03-17 at 18 27 51

### CODE REVIEW

File: apps/web/src/components/digest-demo-flow/DigestDemoFlow.tsx [hunk 1]
```diff
@@ -16,10 +16,14 @@ export function DigestDemoFlow({
   isReadOnly = true,
   templateId,
   className,
+  onRunTriggerClick,
+  onDigestIntervalChange,
```
Comment (LetItRock) [inline apps/web/src/components/digest-demo-flow/DigestDemoFlow.tsx:20]: exposed these functions to the outside in order to track analytics events

File: apps/web/src/pages/templates/editor/DigestWorkflowTourTooltip.tsx [hunk 2]
```diff
@@ -53,6 +55,31 @@ const DotsNavigationStyled = styled(DotsNavigation)`
...
+
+const HINT_INDEX_TO_SKIP_ANALYTICS = {
+  0: DigestWorkflowTourAnalyticsEnum.FIRST_HINT_SKIP_TOUR_CLICK,
+  1: DigestWorkflowTourAnalyticsEnum.SECOND_HINT_SKIP_TOUR_CLICK,
+};
+
+const HINT_INDEX_TO_NAVIGATION_ANALYTICS = {
+  0: DigestWorkflowTourAnalyticsEnum.NAVIGATE_TO_FIRST_HINT_CLICK,
+  1: DigestWorkflowTourAnalyticsEnum.NAVIGATE_TO_SECOND_HINT_CLICK,
+  2: DigestWorkflowTourAnalyticsEnum.NAVIGATE_TO_THIRD_HINT_CLICK,
```
Comment (djabarovgeorge) [inline apps/web/src/pages/templates/editor/DigestWorkflowTourTooltip.tsx:72]: Love this one do you think we can move it to the constants file?

**Review by djabarovgeorge — APPROVED** [review]
Looks great! :), I left a couple of notes there.

**Comment by djabarovgeorge** [comment]
Hey @LetItRock, due @scopsy suggestion https://github.com/novuhq/novu/pull/3031#discussion_r1138778733
I changed a bit the events on your PR to use more the event params so it would be easier to create a 'high level' fannel with one event, so the event funnel creator won't need to collect all the events with conditions.

I was not able to reduce the following events because the first two are 'next' buttons and the last one is 'got it' therefore i left them at the moment.
 DigestWorkflowTourAnalyticsEnum.FIRST_HINT_NEXT_CLICK,
 DigestWorkflowTourAnalyticsEnum.SECOND_HINT_NEXT_CLICK,
 DigestWorkflowTourAnalyticsEnum.THIRD_HINT_GOT_IT_CLICK

**Comment by LetItRock** [comment]
> Hey @LetItRock, due @scopsy suggestion [#3031 (comment)](https://github.com/novuhq/novu/pull/3031#discussion_r1138778733) I changed a bit the events on your PR to use more the event params so it would be easier to create a 'high level' fannel with one event, so the event funnel creator won't need to collect all the events with conditions.
> 
> I was not able to reduce the following events because the first two are 'next' buttons and the last one is 'got it' therefore i left them at the moment. DigestWorkflowTourAnalyticsEnum.FIRST_HINT_NEXT_CLICK, DigestWorkflowTourAnalyticsEnum.SECOND_HINT_NEXT_CLICK, DigestWorkflowTourAnalyticsEnum.THIRD_HINT_GOT_IT_CLICK

thanks @djabarovgeorge ! 🚀

**Pull request merged**